pkg/protostore/protostore.go

```diff
@@ -76,22 +77,34 @@ type Metadata struct {
 }
 
 type ProtoStoreMsg struct {
-	Id         string `gorm:"primaryKey"`
-	Msg        []byte
-	ParentId   string
-	Revision   int64
-	OrgId      string `gorm:"primaryKey"`
-	InstanceId string `gorm:"primaryKey"`
-	CreatedAt  time.Time
-	UpdatedAt  time.Time
-	DeletedAt  gorm.DeletedAt
-	XTableName string `gorm:"-"`
+	Id         string         `gorm:"primaryKey" json:"id"`
+	Msg        []byte         `json:"-"`
+	ParentId   string         `json:"parent_id,omitempty"`
+	Revision   int64          `json:"revision"`
+	OrgId      string         `gorm:"primaryKey" json:"org_id"`
+	InstanceId string         `gorm:"primaryKey" json:"instance_id,omitempty"`
+	CreatedAt  time.Time      `json:"-"`
+	UpdatedAt  time.Time      `json:"-"`
+	DeletedAt  gorm.DeletedAt `json:"-"`
+	XTableName string         `gorm:"-" json:"x_table_name"`
 }
 
 func (msg *ProtoStoreMsg) TableName() string {
 	return msg.XTableName
 }
 
+func (msg *ProtoStoreMsg) String() string {
+	if msg == nil {
+		return "{}"
+	}
+
+	if bytes, err := json.MarshalIndent(msg, "", "\t"); err != nil {
```

Comment:
using single lines with " " for spacing might be better for logging

Reply:
ok, I'll add that now

Reply:
I've removed indentation, making the JSONs printed on a single line. This is what a JSON looks like now:
```json
{"id":"P4","revision":0,"org_id":"Pepsi","instance_id":"Americas","x_table_name":"processor"}
```

The `Marshall()` method that I'm using now doesn't put spaces between the JSON fields. `encoding/json` package doesn't have a way to configure so that some custom string (such as space) is included between fields.